src/pages/index.js

```diff
@@ -0,0 +1,302 @@
+import "../pages/index.css";
+import "../vendor/fonts.css";
+import {
+  enableValidation,
+  settings,
+  showInputError,
+  hideInputError,
+  checkInputValidity,
+  hasInvalidInput,
+  toggleButtonState,
+  disableButton,
+  setEventListeners,
+  resetValidation,
+} from "../scripts/validation.js";
+import avatarSrc from "../images/avatar.jpg";
+import pencilIcon from "../images/pencil-light.svg";
+import logo from "../images/spots_logo.svg";
+import Api from "../utils/Api.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a0ff7229-663f-4e80-b652-bd3a7309c542",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
```

Comment:
`getAppInfo`


You have the `getInitialCards` method for getting only the cards. So, you need to use it in App instead of `getAppInfo`


```js
api
  .getInitialCards()
  .then((cards) => {
    cards.forEach((item) => {
      const cardEl = createCard(item);
      cardsList.append(cardEl);
    });
  })
  .catch((err) => {
    console.error(err);
  });


```